### DISCUSSION TITLE
错误信息：无法连接qbittorrent下载器，userid=None

### DISCUSSION THREAD

**Discussion by arabicman** [jxxghp/MoviePilot#2592] [讨论]
【ERROR】2024-07-21 01:20:18,327 - download.py - 谈判专家 (2024) 添加下载任务失败：Crisis Negotiators 2024 2160p WEB-DL HEVC 60FPS AAC 2.0-HDKWeb - https://www.hdkyl.in/download.php?id=39930，无法连接qbittorrent下载器
【INFO】2024-07-21 01:20:18,329 - chain - 发送消息：channel=None，title=添加下载任务失败：谈判专家 (2024) , text=站点：麒麟
种子名称：Crisis Negotiators 2024 2160p WEB-DL HEVC 60FPS AAC 2.0-HDKWeb
错误信息：无法连接qbittorrent下载器，userid=None

**Comment by arabicman** [comment]
【ERROR】2024-07-21 01:24:16,378 - qbittorrent.py - qbittorrent 连接出错：Failed to connect to qBittorrent. Connection Error: ConnectionError(MaxRetryError("HTTPConnectionPool(host='127.0.0.1', port=8080): Max retries exceeded with url: /api/v2/auth/login (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fee027bba90>: Failed to establish a new connection: [Errno 111] Connection refused'))"))

**Comment by arabicman** [comment]
解决方案：重启qbit容器

**Reply by arabicman** [reply]
问题解决

**Reply by gzsxy** [reply]
我重启容器依然存在问题。不知道为什么了
